src/components/MultiHopTrade/MultiHopTrade.tsx

```diff
@@ -189,7 +189,7 @@ const TradeRoutes = memo(({ isCompact, isStandalone, onChangeTab }: TradeRoutesP
   const quoteListElement = useMemo(
     () => (
       <SlideTransitionRoute
-        height={tradeInputRef.current?.offsetHeight ?? '500px'}
+        height={tradeInputRef.current?.offsetHeight ?? '660px'}
```

Comment:
Mmmh, it smells a little bit, why don't we apply the same styling to the standalone swapper quote list? 

![image](https://github.com/user-attachments/assets/0a1515a7-0bb0-47f1-80b0-97fbdaaabb8f)
![image](https://github.com/user-attachments/assets/18e11389-c90a-4ad3-b3f7-8b7a7b4c8f4b)

Let's accept this as a quickfix but I think we should somehow make it the same height than the swapper one to avoid some content shifts

Reply:
Nice catch ser! Got rugged by diff which is why it was happy in Jam, pushed in 93d8e98434bd934003cb674e6020e9b6680404bf https://jam.dev/c/5f867d44-118f-4714-a75b-97fccefe6e44